n/neo4j/neo4j-5.9.0-ubi8.7.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#---------------------------------------------------------------------------------------------------
+#
+# Package		: Neo4j
+# Version		: 5.9.0
+# Source repo		: https://github.com/neo4j/neo4j.git
+# Tested on		: UBI 8.7 (docker)
+# Language		: Java
+# Travis-Check		: true
+# Script License	: Apache License, Version 2 or later
+# Maintainer		: Sumit Dubey <sumit.dubey2@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#---------------------------------------------------------------------------------------------------
+
+set -eux
```

Comment:
Not sure if we want to enable debugging, no harm done I guess, but we typically just use the "-e" flag and that too on the first line